README.md

```diff
@@ -66,7 +66,12 @@ Since 1.1.0:
 * `quarkus.tools.formatting.surroundEqualsWithSpaces` : Insert spaces around the equals sign when formatting the application.properties file. Default is `false`.
 * `quarkus.tools.validation.required.severity` : Validation severity for required Quarkus properties.
 Default is `none`.
-        
+
+## Articles
+
+ * [How the new Quarkus extension for Visual Studio Code improves the development experience](https://developers.redhat.com/blog/2019/09/23/how-the-new-quarkus-extension-for-visual-studio-code-improves-the-development-experience/)
```

Comment:
https://quarkus.io/blog/vscode-quarkus-1.1.0/ is missing. I don't think it was cross posted to the RH blog. To maintain consistency, you should prolly link the other articles to the quarkus.io urls

Reply:
@fbricon updated